src/core/polyhedral/codegen_llvm.cc

```diff
@@ -449,6 +467,57 @@ class LLVMCodegen {
     }
   }
 
+  void createWrapper(
+      const std::vector<Halide::ImageParam>& inputs,
+      const std::vector<Halide::OutputImageParam>& outputs,
+      const std::string& fname) {
+    CHECK(not inputs.empty());
```

Comment:
can you use something like `CHECK_NE`? this is looking like python atm

Reply:
yep, that's why it looks like python because of `not`. With your explanation, I understand better now the choice for `not`. Thanks :)